src/components/DiveWish.spec.js

```diff
@@ -34,6 +36,16 @@ describe('DiveWish', () => {
     expect(toggleBookmark).toHaveBeenCalled();
   });
 
+  it('clicking the edit button calls the function to edit the dive wish', () => {
```

Comment:
just a suggestion for better reading...it(calls the function to edit the dive wish when the button is clicked), also in line 49

Reply:
Thx